src/ray/common/bundle_spec.cc

```diff
@@ -62,6 +62,20 @@ const ResourceRequest &BundleSpecification::GetRequiredResources() const {
   return *unit_resource_;
 }
 
+absl::flat_hash_map<std::string, double> BundleSpecification::GetWildcardResources()
```

Comment:
Please add an unit test!

Reply:
This function will be removed at @wumuzi520 's PR (#24842).